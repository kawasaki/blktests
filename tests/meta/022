#!/bin/bash
# SPDX-License-Identifier: GPL-3.0+
# Copyright (C) 2025 Western Digital Corporation or its affiliates.
#
# Test skip in device_requries() for test_device_array()

. tests/meta/rc

DESCRIPTION="skip test_device_array() in device_requries()"

device_requires() {
	SKIP_REASONS+=("(╯°□°)╯︵ $TEST_DEV ┻━┻")
}

test_device_array() {
	echo '¯\_(ツ)_/¯'
}
