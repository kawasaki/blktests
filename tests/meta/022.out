Running meta/022
Test complete
